user/super/com/google/gwt/emul/java/util/stream/Collectors.java

```diff
@@ -307,6 +337,11 @@ public static <T> Collector<T,?,List<T>> toList() {
     return toCollection(ArrayList::new);
   }
 
+  public static <T> Collector<T, ?, List<T>> toUnmodifiableList() {
```

Comment:
The unmodifiable methods belong in the java 10 patch, with the tests for these methods.